packages/RichTextEditing-Core.package/TextStructure.class/instance/couldDeriveFromPrettyPrinting.st

```diff
@@ -0,0 +1,4 @@
+testing
+couldDeriveFromPrettyPrinting
+
+	^ self attributes allSatisfy: [:each | each couldDeriveFromPrettyPrinting]
```

Comment:
What does this do?

Reply:
This method is also implemented by `TextAttribute` and indicates if the TextAttribute has functional text attributes or if it could come from some kind of pretty printer.